packages/react-scripts/template/README.md

```diff
@@ -207,50 +207,24 @@ To configure the syntax highlighting in your favorite text editor, head to the [
 
 ## Displaying Lint Output in the Editor
 
->Note: this feature is available with `react-scripts@0.2.0` and higher.
+>Note: this feature is available with `react-scripts@0.2.0` and higher.  
+>It also only works with npm 3 or higher.
 
 Some editors, including Sublime Text, Atom, and Visual Studio Code, provide plugins for ESLint.
 
 They are not required for linting. You should see the linter output right in your terminal as well as the browser console. However, if you prefer the lint results to appear right in your editor, there are some extra steps you can do.
 
-You would need to install an ESLint plugin for your editor first.
-
->**A note for Atom `linter-eslint` users**
-
->If you are using the Atom `linter-eslint` plugin, make sure that **Use global ESLint installation** option is checked:
-
-><img src="http://i.imgur.com/yVNNHJM.png" width="300">
-
-
->**For Visual Studio Code users**
-
->VS Code ESLint plugin automatically detects Create React App's configuration file. So you do not need to create `eslintrc.json` at the root directory, except when you want to add your own rules. In that case, you should include CRA's config by adding this line:
-
->```js
->{
->  // ...
->  "extends": "react-app"
->}
->```
-
-Then add this block to the `package.json` file of your project:
+You would need to install an ESLint plugin for your editor first. Then, add a file called `.eslintrc` to the project root:
```

Comment:
I'm cool with changing to whatever but I have a hunch some editor plugins only know about `.eslintrc` and don't read other files. Happy to change to anything that's supported by most IDE plugins though!